.pending/improvements/gaiarest/4065-http-200-with-e

```diff
@@ -0,0 +1 @@
+4065-http-200-with-empty-results
```

Comment:
This should start with `#2007` and contain and contain an explanation on _what_ has been fixed (not the _how_ or _why_)

Reply:
Ahhh, noted! Thanks.
Updated